spec/compiler/formatter/formatter_spec.cr

```diff
@@ -812,7 +812,7 @@ describe Crystal::Formatter do
       end
       CRYSTAL
       def foo(x,
-              y)
+              y,)
```

Comment:
This one is a bit weird IMO, but formatter with `def_trailing_comma` also produces this output, so prob best to leave it as its own bug/enhancement.